node_monitor/node_provider_db.py

```diff
@@ -86,6 +87,17 @@ class NodeProviderDB:
     """
     table_node_label_lookup_cols = ['node_id', 'node_label']
 
+    # TABLE node_provider_lookup
+    # This table will keep up to date with the node providers registered
+    # on the ic api.
+    create_table_node_provider_lookup = """
+        CREATE TABLE IF NOT EXISTS node_provider_lookup (
+            node_provider_id TEXT PRIMARY KEY,
+            node_provider_name TEXT
+        );
+    """
+    table_node_provider_lookup_cols = ['node_provider_id', 'node_provider_name']
+
```

Comment:
Why are we creating a new table? I thought the idea was just to add them to the `subscribers` table but leave the notify box unchecked.

This just seems redundant. We're just copying the ic-api to our own database at this point. For what reason?
I thought this PR was to make it easier to add new node providers by clicking a box? Why are we just recording node providers at this point?

Reply:
Currently, the `subscribers` table practically functions as the Node Provider table (if all NPs get added to Node Monitor). The idea behind creating a separate table is to have this lookup table, so that `subscribers` can be turned into a many-to-many table. 

This way, you can have different people monitoring the same node provider principal but with different preferences. The can be ideal for having a catch-all channel or even just for NPs that have multiple members on their team and their team would like to be able to set their own preferences for where they get notified.